pkg/server/service.go

```diff
@@ -44,6 +48,7 @@ var defaultCorsHeaders = []string{"Content-Type"}
 func Serve(ctx context.Context, pluginRegistry *plugins.Registry, additionalHandlers map[string]func(http.ResponseWriter, *http.Request)) error {
 	serverConfig := config.GetConfig()
 	configuration := runtime2.NewConfigurationProvider()
+	SetMetricKeys(configuration.ApplicationConfiguration().GetTopLevelConfig())
```

Comment:
Will this cause an issue with single binary? or does it not call Serve? we will be calling SetMetric keys multiple times now..

Reply:
it does call serve but currently sets the metric keys in an init() which i was going to update in a subsequent pr pulling in this admin release https://github.com/flyteorg/flyte/blob/3642ae82fea0d244a0bb3ece36838225ef2109ee/cmd/single/start.go#L189

Reply:
What will you update the single binary to do?

Reply:
i'll update it to pick up the admin release and remove the above snippet setting labeled metric keys in init() and use the config instead